letsencrypt/rootfs/etc/cont-init.d/file-structure.sh

```diff
@@ -34,6 +34,16 @@ echo -e "dns_cloudxns_api_key = $(bashio::config 'dns.cloudxns_api_key')\n" \
       "aws_access_key_id = $(bashio::config 'dns.aws_access_key_id')\n" \
       "aws_secret_access_key = $(bashio::config 'dns.aws_secret_access_key')\n" \
       "dns_sakuracloud_api_token = $(bashio::config 'dns.sakuracloud_api_token')\n" \
-      "dns_sakuracloud_api_secret = $(bashio::config 'dns.sakuracloud_api_secret')" > /data/dnsapikey
+      "dns_sakuracloud_api_secret = $(bashio::config 'dns.sakuracloud_api_secret')\n" \
+      "certbot_dns_transip:dns_transip_username = $(bashio::config 'dns.transip_username')\n" \
+      "certbot_dns_transip:dns_transip_key_file = /data/transip-rsa.key\n" > /data/dnsapikey
```

Comment:
Minor detail, but the `\n` after `/data/transip-rsa.key` is probably not required as there was not one before.